doc/modules/impute.rst

```diff
@@ -105,9 +105,12 @@ imputation round are returned.
 
 .. note::
 
-   This estimator is still **experimental** for now: the predictions
-   and the API might change without any deprecation cycle. To use it,
-   you need to explicitly import ``enable_iterative_imputer``.
+   This estimator is still **experimental** for now: default parameters or
+   details of behaviour might change without any deprecation cycle,
+   specifically convergence criteria (:issue:`14338`),
+   default estimators (:issue:`13286`),
+   and use of random state (:issue:`15611`).
+   To use it, you need to explicitly import ``enable_iterative_imputer``.
```

Comment:
This can be broken up into two sentences:

> This estimator is still experimental for now: default parameters or details of behavior might change without any deprecation cycle. Resolving the following issues would help stabilize :class:`IterativeImputer`: ...

Reply:
Updated.